.travis.yml

```diff
@@ -8,15 +8,36 @@ sudo: false
 addons:
   apt:
     packages:
+    - bc
+    - doxygen
+    - expect
+    - gettext
+    - libncurses5-dev
     - tree
-    - fish
+
+env:
+  global:
+    - PATH="$HOME/fish/bin:$PATH"
+    - OMF_REPO_URI="https://github.com/$TRAVIS_REPO_SLUG"
+    - OMF_REPO_BRANCH="$TRAVIS_BRANCH"
+
+before_install:
+  - tests/install_fish.sh
+  - fish -c 'ln -sf $TRAVIS_BUILD_DIR/tests/fish_user_function $fish_function_path[1]'
     
 before_script: pwd; tree -h
 
-script: /bin/sh bin/install
+script:
+  - /bin/sh tests/travis_install.sh
+  - tests/print_prompt
```

Comment:
About the` tests/print_prompt`, there wasn't any goal here.
Just wanted to see if the `print_prompt` behave correctly and kind of let it here.